x/sigs/handler.go

```diff
@@ -0,0 +1,54 @@
+package sigs
+
+import (
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/orm"
+)
+
+func RegisterRoutes(r weave.Registry) {
+	r.Handle(pathBumpSequenceMsg, &bumpSequenceHandler{
+		b: NewBucket(),
+	})
+}
+
+type bumpSequenceHandler struct {
+	b Bucket
+}
+
+func (h *bumpSequenceHandler) Check(ctx weave.Context, db weave.KVStore, tx weave.Tx) (weave.CheckResult, error) {
+	_, _, err := h.validate(ctx, db, tx)
+	return weave.CheckResult{}, err
+}
+
+func (h *bumpSequenceHandler) Deliver(ctx weave.Context, db weave.KVStore, tx weave.Tx) (weave.DeliverResult, error) {
+	user, msg, err := h.validate(ctx, db, tx)
+	if err != nil {
+		return weave.DeliverResult{}, err
+	}
+
+	user.Sequence += msg.Increment
```

Comment:
req: you need to prevent an overflow here. see [sequence type](https://github.com/iov-one/weave/blob/master/x/sigs/codec.proto#L14) better move the check to validation.